### ISSUE TITLE
Suggestions to make it easier/better

### ISSUE THREAD

**Issue by simonm3** [rkhwaja/fs.googledrivefs#23]
Really useful package thanks! A few suggestions to make better:

1) Please include install instructions
2) Can you change the logging so it uses a different logger or logs to debug. Currently it uses root logger at info level so if you do hundreds of calls you get a lot of log messages and no way to silence them.
3) Many people using google drive are primarily trying to access their own account. Would be great if you could make this simpler as the oauth2 docs are a bit wordy and not very clear. I have written some code for this leveraging pydrive which you could use as a base.
`
from pydrive.auth import GoogleAuth
import logging
import json
from configparser import ConfigParser
from os.path import expanduser
HOME = expanduser("~")

logging.getLogger("googleapiclient").setLevel(logging.ERROR)
logging.getLogger("oauth2client").setLevel(logging.ERROR)

def gdrive(force=False, credsfile=f"{HOME}/.gdrive.json"):
    """ return credentials for google drive

    onetime setup::
        pip install fs.googledrivefs
        enable google drive api
        create creds
            credentials/create oauth clientid
            select web application
            authorised javascript origins http://localhost:8080
            authorised redirect urls http://localhost:8080/ [NOTE THE / on the end]
            download client_secrets.json and move to ~/.gdrive.json

    Usage to open google drive using pyfilesystem::
        fs1 = fs.open_fs(f"googledrive://{gdrive()}")

    :param force: force reauthentication. needed if token expired or cancelled.
    :param credsfile: location of client_secrets.json downloaded from google
    :return: authorisation string for google drive
    """
    # get creds that are needed to obtain tokens
    creds = json.load(open(credsfile))
    web = creds["web"]
    client_id = web["client_id"]
    client_secret = web["client_secret"]

    # get tokens once per device and save in credsfile
    if force or "refresh_token" not in web:
        gauth = GoogleAuth()
        gauth.settings["client_config_file"] = credsfile
        gauth.settings["get_refresh_token"] = True
        gauth.LocalWebserverAuth()
        web["access_token"] = gauth.credentials.access_token
        web["refresh_token"] = gauth.credentials.refresh_token
        json.dump(creds, open(credsfile, "w"))
    access_token = web["access_token"]
    refresh_token = web["refresh_token"]

    return f"?client_id={client_id}&client_secret={client_secret}&" \
           f"access_token={access_token}&refresh_token={refresh_token}"`

**Comment by rkhwaja** [comment]
Closing, I addressed your first 2 points. For the last, I'm not sure where such code belongs.
